spyderlib/widgets/editor.py

```diff
@@ -581,10 +581,16 @@ def open_fileswitcher_dlg(self):
         """Open file list management dialog box"""
         if not self.tabs.count():
             return
+        if self.fileswitcher_dlg is not None and \
+          self.fileswitcher_dlg.is_visible:
+            self.fileswitcher_dlg.hide()
+            self.fileswitcher_dlg.is_visible = False

```

Comment:
It is still present in Qt5: http://doc.qt.io/qt-5.6/qwidget.html#visible-prop (and QDialog inherits QWidget). So it should work...
Anyway it's not a big deal, this comment is for completeness only.